test/functional/testlib.bash

```diff
@@ -1377,6 +1380,12 @@ destroy_test_environment() { # swupd_function
 		return 1
 	fi
 
+	if [ "$DEBUG_TEST" = true ]; then
```

Comment:
We still have one problem. If the bats file has 2 tests the second will remove the env from the first. 
Can't we create different envs for each test in the bats file?
like ${TEST_NAME}_1 or something similar?

Reply:
yep, sounds like we could create the environment with something like this:
`"$TEST_NAME"_"$BATS_TEST_NUMBER"`